sktime/transformations/series/compose.py

```diff
@@ -96,3 +97,143 @@ def inverse_transform(self, Z, X=None):
+    columns : list of str
+            Names of columns that are supposed to be transformed
+    """
+
+    _required_parameters = ["transformer"]
+
+    def __init__(self, transformer, columns="all"):
+        self.transformer = transformer
+        self.columns = columns
+        self.transformers_ = None
+        super(ColumnComposition, self).__init__()
+
+    def fit(self, Z, X=None):
+        """
+        Iterates over columns (series) and applies the fit function of the transformer
+        """
+        z = check_series(Z, allow_numpy=False)
+        self._is_fitted = False
+
+        # univariate case
+        if isinstance(z, pd.Series):
+            self.transformer.fit(z, X)
+            self._is_fitted = True
+            return self
+        # multivariate case
+        else:
+            if self.columns == "all":
+                self.columns = z.columns
+
+            # make sure z contains all columns that the user wants to transform
+            Z_wanted_keys = set(self.columns)
+            Z_new_keys = set(z.columns)
+            difference = Z_wanted_keys.difference(Z_new_keys)
+            if len(difference) != 0:
+                raise ValueError("Missing columns" + str(difference) + "in Z.")
+
+            self.transformers_ = {}
+            for colname in self.columns:
+                transformer = clone(self.transformer)
+                self.transformers_[colname] = transformer
+                self.transformers_[colname].fit(z[colname], X)
+                # self.transformers_[colname].is_fitted = True
+            self._is_fitted = True
+            return self
+
+    def transform(self, Z, X=None):
+        """
+        calls transform on every single transformer (one transformer per series)
+        """
+        self.check_is_fitted()
+        z = check_series(Z)
+
+        # univariate case
+        if isinstance(z, pd.Series):
+            z = self.transformer.transform(z, X)
+            return z
+
+        # multivariate case
+        else:
+            # make copy of z
+            z = z.copy()
+            # make sure z contains all columns that the user wants to transform
+            Z_wanted_keys = set(self.columns)
+            Z_new_keys = set(z.columns)
+            difference = Z_wanted_keys.difference(Z_new_keys)
+            if len(difference) != 0:
+                raise ValueError("Missing columns" + str(difference) + "in Z.")
+            for colname in self.columns:
+                # self.columns : columns that are supposed to be transformed
+                self.transformers_[colname].check_is_fitted()
+                z[colname] = self.transformers_[colname].transform(z[colname], X)
+            return z
+
+    def inverse_transform(self, Z, X=None):
+        """
+        If the base transformer has an inverse-transform this
+        inverse transform is called on every single transformer
+        (one transformer per series)
+        """
```

Comment:
also here, you can just copy the docstring about the params from another transformer